.travis.yml

```diff
@@ -18,4 +18,4 @@ before_install:
   - go get github.com/mattn/goveralls
   - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
 script:
-    - $HOME/gopath/bin/goveralls -service=travis-ci
```

Comment:
Hello. Thanks for the nice PR.

BTW, It does not seem to be completely rolled back. Could you do a complete rollback?

Reply:
completed